openmetadata-ui/src/main/resources/ui/src/components/TableQueries/TableQueries.interface.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { OperationPermission } from 'components/PermissionProvider/PermissionProvider.interface';
+import { Query } from 'generated/entity/data/query';
+import { EntityReference } from 'generated/type/entityReference';
+import { HTMLAttributes } from 'react';
+
+export enum QueryVoteType {
+  'votedUp' = 'votedUp',
+  'votedDown' = 'votedDown',
+  'unVoted' = 'unVoted',
```

Comment:
```suggestion
  VOTED = 'voted',
  DOWN_VOTED = 'down_voted',
  NOT_VOTED = 'not_voted',
```

Reply:
So this is created as per Data which the backend is expecting